officers/liviana_charvanek.json

```diff
@@ -73,7 +73,7 @@
     },
     {
       "name": "Commander V",
-      "officer_ability_value": 10,
+      "officer_ability_value": 0.01,
```

Comment:
I dont think this was intended

Reply:
Absolutely not ;) did the changes via remote desktop from my smart phone. Really not enough space on the display to see everything in the git client. Sorry